api/integrations/launch_darkly/services.py

```diff
@@ -156,21 +310,29 @@ def _create_string_feature_states(
 
         feature_state, _ = FeatureState.objects.update_or_create(
             feature=feature,
+            feature_segment=None,
```

Comment:
Previously, there was only a single `FeatureState`, which allowed running import multiple times. After my changes, multiple `FeatureState` objects belong to this feature because they target a `feature_segment`.

So, we should match for states that do not belong to any segment for this piece of code. This should set the state that we see when we click the feature from UI (the main state)